server/sonar-db-dao/src/main/java/org/sonar/db/webhook/WebhookDao.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.db.webhook;
+
+import java.util.List;
+import java.util.Optional;
+import org.sonar.api.utils.System2;
+import org.sonar.db.Dao;
+import org.sonar.db.DbSession;
+
+import static com.google.common.base.Preconditions.checkState;
+
+public class WebhookDao implements Dao {
+
+  private final System2 system2;
+
+  public WebhookDao(System2 system2) {
+    this.system2 = system2;
+  }
+
+  public Optional<WebhookDto> selectByUuid(DbSession dbSession, String uuid) {
+    return Optional.ofNullable(mapper(dbSession).selectByUuid(uuid));
+  }
+
+  public List<WebhookDto> selectByOrganizationUuid(DbSession dbSession, String organizationUuid) {
```

Comment:
To benefit from typing, I suggest to replace signature with:
`selectByOrganization(DbSession dbSession, OrganizationDto organization)`

That avoids to have only callers doing `selectByOrganizationUuid(..., org.getUuid())`.
Note that keeping String parameters in WebhookMapper is the way to go.

Reply:
According to SQL the results are ordered by name. It deserves to be documented in the javadoc or to appear in the method name.

Reply:
done